hrjobcontract/js/src/job-contract/controllers/form/form-leave.js

```diff
@@ -0,0 +1,38 @@
+define([
+    'job-contract/vendor/fraction',
+    'job-contract/controllers/controllers'
+], function (Fraction, controllers) {
+    'use strict';
+
+    controllers.controller('FormLeaveCtrl',['$scope', '$rootScope', '$filter', '$log', '$q', 'UtilsService',
+      function ($scope, $rootScope, $filter, $log, $q, UtilsService) {
+        $log.debug('Controller: FormLeaveCtrl');
+
+        var promiseNumberOfPublicHolidays = UtilsService.getNumberOfPublicHolidaysInCurrentPeriod();
+
+        promiseNumberOfPublicHolidays.then(function(result){
+          var number = parseFloat(result);
+          if(isNaN(number)) {
+            number = 0;
+          }
+          $scope.numberOfPublicHolidays = number;
+        });
+
+        $scope.$watch('entity.leave', function(newValue, oldValue) {
+          var newTypeWithPublicHolidays = null;
+          newValue.forEach(function(value, i) {
```

Comment:
Unfortunately the talk about a CI server is on hold for now, can't do much about it